frontend/src/metabase/search/components/filters/CreatedByFilter/CreatedByContent.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import { isEqual } from "underscore";
+import { t } from "ttag";
+import { useUserListQuery } from "metabase/common/hooks/use-user-list-query";
+import type { UserListResult } from "metabase-types/api";
+import { Center, Loader, Text, TextInput } from "metabase/ui";
+import {
+  CreatedByContainer,
+  CreatedByContentContainer,
+} from "metabase/search/components/filters/CreatedByFilter/CreatedByContent.styled";
+import type { SearchSidebarFilterComponent } from "metabase/search/types";
+import { UserListElement } from "metabase/search/components/filters/CreatedByFilter/UserListElement";
+
+export const CreatedByContent: SearchSidebarFilterComponent<"created_by">["ContentComponent"] =
+  ({ value, onChange }) => {
+    const { data: users = [], isLoading } = useUserListQuery();
+    const [userFilter, setUserFilter] = useState("");
+
+    const filteredUsers = users.filter(user =>
+      user.common_name.toLowerCase().includes(userFilter.toLowerCase()),
+    );
```

Comment:
1. Create a user named `Óisín Coveney`
2. Log in as this user and create a question called "XYZ"
3. Search for "XYZ" and click "View and filter all 1 results"
4. Notice that we have a bug here and the label should be different when there is only 1 result, or it should be removed altogether (there is no reason to filter 1 result) :smile: 
5. Click "Creator" input and focus the search input in the popover
6. Type `ois`

Expected result: `Óisín Coveney` is found
Actual result: No users found

This applies to all non-basic-latin characters (`Kąmił Mięłnik` - PL, `John İstanbul` - TR, etc.).

(BTW it's a good test case scenario)

Reply:
Hm... this does raise a really good question. Personally, I think the normalization/deburring/(whatever you want to call removing accents/etc) logic should be in the backend when we're searching.

As for the CTA label, I've just solved that on the 1.5-integration branch (which this PR is based on), so it won't be an issue on master once that's merged.

Reply:
I actually might be able to add that logic on the backend, I'll try it out and ask one of the BEs to verify